.github/workflows/release.yaml

```
@@ -192,6 +192,19 @@ jobs:
         env:
           GH_TOKEN: ${{ github.token }}
 
+      - name: Login to the OCI registries
+        run: |
+          echo "${{ secrets.QUAY_DEPLOYER_PASSWORD }}" | helm registry login quay.io --username ${{ secrets.QUAY_DEPLOYER_USERNAME }} --password-stdin
+          echo "${{ secrets.DOCKER_PASSWORD }}" | helm registry login docker.io --username ${{ secrets.DOCKER_USERNAME }} --password-stdin
+          echo "${{ github.token }}" | helm registry login ghcr.io --username ${{ github.actor }} --password-stdin
+
+      - name: Push helm chart to the OCI registries
+        run: |
+          release_version=$(./tools/packaging/release/release.sh release-version)
+          helm push kata-deploy-${release_version}.tgz oci://quay.io/kata-containers/kata-deploy-charts
+          helm push kata-deploy-${release-version}.tgz oci://docker.io/katadocker/kata-deploy-charts
+          helm push kata-deploy-${release-version}.tgz oci://ghcr.io/kata-containers/kata-deploy-charts
```

I think action-lint wants this change:
```suggestion
          helm push "kata-deploy-${release_version}.tgz" oci://quay.io/kata-containers/kata-deploy-charts
          helm push "kata-deploy-${release-version}.tgz" oci://docker.io/katadocker/kata-deploy-charts
          helm push "kata-deploy-${release-version}.tgz" oci://ghcr.io/kata-containers/kata-deploy-charts
```